hr_employee_id/views/res_config_views.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="view_human_resources_employeeid_configuration" model="ir.ui.view">
+        <field name="name">view.human.resources.employeeid.configuration</field>
+        <field name="model">hr.employeeid.config.settings</field>
+        <field name="arch" type="xml">
+            <form string="Configure Employee ID" class="oe_form_configuration">
+                <header>
+                    <button string="Apply" type="object" name="execute" class="oe_highlight"/>
+                    <button string="Cancel" type="object" name="cancel" class="oe_link"/>
+                </header>
+                <group>
+                    <label for="id" string="Employee ID *"/>
+                    <div>
+                        <div>
+                            <label for="employee_id_gen_method"/>
+                            <field name="employee_id_gen_method" class="oe_inline"/>
+                        </div>
+                        <div attrs="{'invisible': [('employee_id_gen_method','!=','random')]}">
+                            <label for="employee_id_random_digits"/>
+                            <field name="employee_id_random_digits" class="oe_inline"
+                                attrs="{'required': [('employee_id_gen_method','=','random')]}"/>
+                        </div>
+                        <div
+                            attrs="{'invisible': [('employee_id_gen_method','!=','sequence')]}">
+                            <label for="employee_id_sequence"/>
+                            <field name="employee_id_sequence" class="oe_inline"
+                                attrs="{'required': [('employee_id_gen_method','=','sequence')]}"/>
+                        </div>
+                    </div>
+                </group>
+                <div>
+                    <span>(*) This configuration is related to the company you're logged into.</span>
+                </div>
+            </form>
+        </field>
+    </record>
+
+    <record id="action_human_resources_employeeid_config" model="ir.actions.act_window">
+        <field name="name">Configure Employee ID</field>
+        <field name="type">ir.actions.act_window</field>
+        <field name="res_model">hr.employeeid.config.settings</field>
+        <field name="view_mode">form</field>
+        <field name="target">inline</field>
+    </record>
+
+    <menuitem id="menu_account_config" name="Employee ID" parent="hr.menu_human_resources_configuration"
```

Comment:
@pedrobaeza this menu :)

Reply:
Well, I figure out that in the mind of original developers, the configuration is quite a technical action, so they hide if you don't have technical permissions (which is in vanilla Odoo activated when on debug mode), but if you hate as most of us the technical mode (and enable it by default to all), you can remove it.

Reply:
Well, from a usability point of view, we have system administrators and specific Head of Staff, and calling sysadmins to enable debug mode for configuration is quite-quite odd. I think I'll remove it

Reply:
OK, it's something nobody will miss for sure.